packages/api/src/routes/v3/report.ts

```diff
@@ -59,11 +63,13 @@ export default (server: FastifyInstance, _options: RouteShorthandOptions, done:
             `- <b>URL:</b> ${request.body.url}`,
             `- <b>User-Agent:</b> ${request.body.userAgent ?? '-'}`,
             `- <b>Version:</b> ${request.body.version}`,
+            `- <b>Explanation:</b> ${request.body.explanation ? request.body.explanation : '-'}`,
```

Comment:
I would rather do something like `- <b>Explanation:</b> ${request.body.explanation || '-'}` in here